tools/tpm2_quote.c

```diff
@@ -261,19 +261,39 @@ static int quote(TSS2_SYS_CONTEXT *sapi_context, TPM_HANDLE akHandle, TPML_PCR_S
         printf("OutFile: %s Can Not Be Created !\n",outFilePath);
         return -2;
     }
+
+    FILE *sigfp = NULL;
+    if(s_flag)
+    {
```

Comment:
Please check the coding style at misc/coding_standard_c.txt and follow that.

Reply:
Fixed